src/styles/main.scss

```
@@ -1,3 +1,57 @@
+@import "./utils/variables";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  width: $calendar-width;
+  padding: $calendar-padding;
+  @each $day, $margin-left in $start-day {
+    &--start-day--#{$day} {
+      .calendar__item:first-child {
+        margin-left: $margin-left;
+      }
+    }
+  }
+  @for $index from 28 through 31 {
```

```suggestion
  @for $days-in-month from 28 through 31 {
```
use more meaningful names for variables, for example `days-in-month`